external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -591,7 +589,7 @@ public String toString() {
 
     // =========== Timer ===========
 
-    private class Timer {
+    public static class Timer {
```

Comment:
I think that we should make Timer it's own class, rather than a static inner class, since it's used in other objects as well.